src/modules/ProductCard/ProductCard.module.scss

```diff
@@ -0,0 +1,139 @@
+.card {
+  width: 272px;
+  height: 506px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 32px;
+  gap: 8px;
+
+  position: absolute;
+  left: 16px;
+  top: 16px;
+
+  background: #161827;
+
+  &_image {
+    height: 196px;
+    width: 208px;
+
+    display: block;
+    margin-bottom: 24px;
+  }
+
+  &_name {
+    font-family: 'Mont-Regular', sans-serif;;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 14px;
+    line-height: 21px;
+    margin-bottom: 8px;
+
+    color: $color-white;
+  }
+
+  &_price {
+    font-family: 'Mont-Bold', sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 22px;
+    line-height: 31px;
+    text-align: center;
+    margin-bottom: 8px;
+
+    color: $color-white;
+  }
+
+  &_divide-line {
+    width: 100%;
+    height: 1px;
+
+    margin-bottom: 16px;
+    border: 1px solid #3B3E4A;
+  }
```

Comment:
It's better to do this element like ::before(::after) element with position absolute etc. But I think we can stay with this variant, because now I don't understand which element should be a parent with position relative. So without change for now